pkgs/development/python-modules/torch/default.nix

```diff
@@ -143,10 +144,18 @@ in buildPythonPackage rec {
     repo = "pytorch";
     rev = "refs/tags/v${version}";
     fetchSubmodules = true;
-    hash = "sha256-xUj77yKz3IQ3gd/G32pI4OhL3LoN1zS7eFg0/0nZp5I=";
+    hash = "sha256-01+uqHvPbQVXKLohGWfsCsZOjb7xmfjBKkTGUGMEdAI=";
   };
 
-  patches = lib.optionals (stdenv.isDarwin && stdenv.isx86_64) [
+  prePatch = lib.optionalString cudaSupport ''
+    rm cmake/Modules/FindCUDAToolkit.cmake
+  '';
```

Comment:
Actually, we already got a `postPatch`, why don't we move it over there (and do we ever need `prePatch` other than when `patches` fail to apply without some prep?)

Reply:
I moved it to `postPatch`.